### PR TITLE
docs: Updated doc to comply with 1.0.0-alpha.8 release of TS client

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/client-libraries/typescript.md [hunk 1]
```diff
@@ -284,10 +265,7 @@ db.transact(async (tx) => {
   // add balance to user2
   await users.update(
     {
-      op: SelectorFilterOperator.EQ,
-      fields: {
-        userId: user2.userId,
-      },
+      userId: user2.userId,
     },
```
[the remark this answers is not in the document]
Comment (JigarJoshi) [reply docs/client-libraries/typescript.md:269]: sure.

[the remark this answers is not in the document]
Comment (JigarJoshi) [reply docs/client-libraries/typescript.md:269]: updated.

File: docs/client-libraries/java/async-client.md [hunk 2]
```diff
@@ -195,3 +195,9 @@ db
         }
     });
 ```
+
+## Drop database
+
+```java
+asyncClient.dropDatabase("sampledb").get();
```
Comment (ovaistariq) [inline docs/client-libraries/java/async-client.md:202]: Do we need `get()` at the end?

File: docs/client-libraries/typescript.md [hunk 3]
```diff
@@ -135,10 +134,7 @@ documents matching the filtering condition are fetched.
 ```typescript
 users
   .readOne({
-    op: SelectorFilterOperator.EQ,
-    fields: {
-      userId: 1,
-    },
+    userId: 1,
   })
   .then((user) => {
```
Comment (ovaistariq) [inline docs/client-libraries/typescript.md:139]: The `.then((user)) ...` doesn't look good in these examples. Can we change all such occurrences to be of this form:

```
user = await users.readOne ...
```

Comment (ovaistariq) [reply docs/client-libraries/typescript.md:139]: Take a look at sequelize examples here: https://sequelize.org/docs/v6/core-concepts/model-querying-basics/

File: docs/client-libraries/java/async-client.md (hunk 2 above)
Comment (JigarJoshi) [reply docs/client-libraries/java/async-client.md]: asyncClient.dropDatabase("sampledb") returns a future object. .get(); will block call until that future is resolved.

File: docs/client-libraries/typescript.md (hunk 3 above)
Comment (JigarJoshi) [reply docs/client-libraries/typescript.md]: updated.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**